src/image_bios.cpp

```diff
@@ -433,6 +446,25 @@ void BIOSUpdater::doInstall(const fs::path& file)
             throw FwupdateError("Unable to preserve x722 MAC: %s", ex.what());
         }
 #endif // INTEL_X722_SUPPORT
+
+#ifdef GOLDEN_FLASH_SUPPORT
+        if (useGoldenFlash)
+        {
+            Tracer tracer("Switching to golden flash");
+
+            // This makes the BMC to reinit the driver which might be attached
+            // with wrong data on the hosts equipped with em100.
+            unbindSPIDriver();
+
+            // Swap ChipSelect of main and golden flash drives
+            gpioActiveFlashSel.set_value(1);
```

Comment:
It seems to me, it would be better to define enum class where CommonFlash=0, and GoldenFlash=1, to avoid magic numbers and provide clear gpio-level purpose.

But just for suggestion,  I don't insist :)

Reply:
fixed